src/main/java/com/kodilla/ecommercee/mapper/UserMapper.java

```diff
@@ -3,18 +3,34 @@
 import com.kodilla.ecommercee.dto.UserDto;
 import com.kodilla.ecommercee.entity.User;
 import org.springframework.stereotype.Component;
-
-import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
 
 @Component
 public class UserMapper {
 
-    public User translateToUser(UserDto userDto) {return new User(); }
+    public User translateToUser(final UserDto userDto) {
```

Comment:
Proponuje zostawić prostszą formę. toUser albo to UserDto bez słówka translate

Reply:
ok